_data/projects/moviejs.yml

```diff
@@ -0,0 +1,12 @@
+name: movie.js
+desc: A flexible video-editing library for the browser
+site: https://github.com/clabe45/movie.js
+tags:
+- javascript
+- vanilla-javascript
+- video
+- video-editing
+- automation
+upforgrabs:
+  name: help wanted
+  link: https://github.com/clabe45/movie.js/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@clabe45 Please change this to https://github.com/clabe45/movie.js/labels/help%20wanted

Reply:
Ok

Reply:
It's updated now